futsal_app/tests/test_add_absence.py

```diff
@@ -0,0 +1,47 @@
+from datetime import datetime
+from django.test import Client, TestCase
+from django.utils import timezone
+from games.models import PlayerStatus, Game, BookingHistoryForGame, Player
+from games import views
+from .base import BaseTestCase
+from django.urls import reverse
+
+
+class SimpleAddAbsenceTest(BaseTestCase):
+    def setUp(self):
+        super().setUp()
+        self.client = Client()
```

Comment:
overriding this method here brings nothing. self.client is already created in the base class